README.md

```diff
@@ -82,28 +74,69 @@ ansible-galaxy collection build --force
 ansible-galaxy collection install cisco-aci-* --force
 ```
 
-### Second Approach
+## Use Cases
+
+Once the collection is installed, you can use it in a playbook by specifying the full namespace path to the module, plugin and/or role.
+
+### Adding a new EPG
 
-Go to [ansible-aci Actions](https://github.com/CiscoDevNet/ansible-aci/actions)
+```yml
+- hosts: aci
+  gather_facts: no
 
-Select the latest CI build
+  tasks:
+  - name: Add a new EPG
+    cisco.aci.aci_epg:
+      hostname: apic
+      username: admin
+      password: SomeSecretPassword
+      tenant: production
+      ap: intranet
+      epg: web_epg
+      description: Web Intranet EPG
+      bd: prod_bd
+    delegate_to: localhost
+```
 
-Under Artifacts download collection and unzip it using Terminal or Console.
+## Optimizing Playbooks
 
-*Note: The collection file is a zip file containing a tar.gz file. We recommend using CLI because some GUI-based unarchiver might unarchive both nested archives in one go.*
+There are two main methods to optimize the execution of ACI modules in your playbooks.
 
-Install the unarchived tar.gz file
+1. Using the ACI HTTPAPI plugin
+1. Using the `suppress_` options
 
-```sh
-ansible-galaxy collection install cisco-aci-1.0.0.tar.gz —-force
-```
+To find out more about optimizing playbook execution, please refer to the [Optimizing Playbooks](docs/optimizing.md) documentation.
 
-### See Also
+## Testing
 
-- [Ansible Using collections](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html) for more details.
+Integration tests for each module in the cisco.aci collections are executed against the following ACI versions:
 
-## Contributing to this collection
+- 4.2(7s)
+- 5.2(7g)
+- 6.0(2h)
```

Comment:
I think we need to be a bit more vague on versions as I don't want to update this every time we change the setups.
```suggestion
- 4.2
- 5.2
- 6.0
```